doc/developer/logging.rst

```diff
@@ -383,7 +383,8 @@ bgpd
 
 .. frrfmt:: %pBD (struct bgp_dest *)
 
-   Print prefix for a BGP destination.
+   Print prefix for a BGP destination.  When using `--enable-dev-build` include
```

Comment:
If we need to highlight, then double backticks needed.

Reply:
fixed